lib/tether/tether_linux.go

```diff
@@ -126,10 +137,16 @@ func (t *tether) childReaper() error {
 func (t *tether) stopReaper() {
```

Comment:
We do, Start calls the cleanup on its way out and cleanup is the only place where we call this function.